### DISCUSSION TITLE
Can't find rules/policies for RealtimeDB

### DISCUSSION THREAD

**Discussion by Kleysley** [supabase/supabase#1754] [Questions] [answered]
For getting started with Supabase I just wanted to explore some functionality. Suppose I created a table called "MyTable" with two colums:
_______________________
username | isAdmin   |
------------------------

Where do I write  rules for this database to secure it and in what language (SQL?)? Because I couldn't find it on the website or the dashboard. There has to be some sort of rule right? (For example if I wanted to make a rule like "ALLOW READ IF ROW.isAdmin = True")

Thank you for your answers!

**Comment by awalias** [comment, marked as answer]
hey @Kleysley this may be useful https://supabase.io/docs/guides/auth#row-level-security

also there are some video guides available here: https://supabase.io/docs/learn/auth-deep-dive/auth-row-level-security

note that Row Level Policies are currently only available on the REST API and Storage API.

Full RLS for the realtime endpoint itself (`/realtime/v1/`) is a work in progress, however there is initial JWT support here: https://github.com/supabase/realtime#websocket-connection-authorization

**Reply by w3b6x9** [reply]
@awalias spot on!

Just wanted to add that if you're using the Supabase Client, you are connected to `Realtime` via JWT (Supabase Key) authorization by default.

**Reply by awalias** [reply]
hey @Kleysley , @w3b6x9 added some clarification in the README here: https://github.com/supabase/realtime/pull/159/files

**Reply by Kleysley** [reply]
Aight, thank you!
